src/utils/convertIngredient.js

```diff
@@ -0,0 +1,7 @@
+const convertIngredient = (text) => {
```

Comment:
Maybe update the name to `convertTextToIngredientObj` 🤔 It provides more contexts from just reading the function name.